contracts/modules/LoanSettings/LoanSettings.sol

```diff
@@ -29,6 +30,7 @@ contract LoanSettings is State, InterestHandler, LoanSettingsEvents {
         _setTarget(this.getPoolLastInterestRate.selector, target);
         _setTarget(this.getLoanPrincipal.selector, target);
         _setTarget(this.getLoanInterestOutstanding.selector, target);
+        _setTarget(this.migrateLoanParamsList.selector, target);
     }
 
     function setupLoanParams(
```

Comment:
make sure to add onlyGuardian functionality and make it so the protocol or guardian can call to initiate the setup of loan params versus the iToken contract.

Reply:
in _setupLoanParams check to see if the loan token and collateral token are supported rather than being non-zero addresses.

Reply:
this was always permission less. coming from p2p borrow/lend where you could set your own params